src/components/Icon/icons.ts

```diff
@@ -106,6 +109,7 @@ export const icons = {
   LinkedinIn: FaLinkedinIn,
   ListBox: IoMdListBox,
   Loading: VscLoading,
+  Logout: HiLogout,
```

Comment:
```suggestion
  Signout: HiLogout,
```

Reply:
removed this icon instead as it is unused